kythe-multi-file-verifier.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -e
+# Copyright 2020 The Verible Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+usage() {
+  cat <<EOF
+
+Extracts Kythe facts from the given verilog file and runs Kythe verifier on the produced facts.
+EOF
+}
+
+set -o pipefail
+KYTHE_BINDIR="/opt/kythe/tools"
+KYTHE_OUT="./kythe-out"
+
+# The files are expected to be self-contained single-file test cases.
+VERILOG_MULTI_FILE_TEST_FILES="./verilog/tools/kythe/testdata/more_testdata/multi_file_test/*.sv"
```

Comment:
This is fine for now, but in the future, we'll want a way to test multiple independent groups of files.  @ivan444 has some ideas about this.